Mathlib/Algebra/Order/Field/Basic.lean

```diff
@@ -195,36 +174,39 @@ lemma div_lt_div_of_pos_left (ha : 0 < a) (hc : 0 < c) (h : c < b) : a / b < a /
 lemma div_le_div_of_le (hc : 0 ≤ c) (hab : a ≤ b) : a / c ≤ b / c :=
   div_le_div_of_nonneg_right hab hc
 
-theorem div_le_div_right (hc : 0 < c) : a / c ≤ b / c ↔ a ≤ b :=
-  ⟨le_imp_le_of_lt_imp_lt fun hab ↦ div_lt_div_of_pos_right hab hc,
-    fun hab ↦ div_le_div_of_nonneg_right hab hc.le⟩
+@[deprecated div_le_div_iff_of_pos_right (since := "2024-11-12")]
+theorem div_le_div_right (hc : 0 < c) : a / c ≤ b / c ↔ a ≤ b := div_le_div_iff_of_pos_right hc
 
-theorem div_lt_div_right (hc : 0 < c) : a / c < b / c ↔ a < b :=
-  lt_iff_lt_of_le_iff_le <| div_le_div_right hc
+@[deprecated div_lt_div_iff_of_pos_right (since := "2024-11-12")]
+theorem div_lt_div_right (hc : 0 < c) : a / c < b / c ↔ a < b := div_lt_div_iff_of_pos_right hc
 
-theorem div_lt_div_left (ha : 0 < a) (hb : 0 < b) (hc : 0 < c) : a / b < a / c ↔ c < b := by
-  simp only [div_eq_mul_inv, mul_lt_mul_left ha, inv_lt_inv₀ hb hc]
+@[deprecated div_lt_div_iff_of_pos_left (since := "2024-11-13")]
+theorem div_lt_div_left (ha : 0 < a) (hb : 0 < b) (hc : 0 < c) : a / b < a / c ↔ c < b :=
+  div_lt_div_iff_of_pos_left ha hb hc
```

Comment:
I'm very confused about what I'm reading here. You deprecate a lemma, but then you use it to prove the lemma which is supposed to take it's place? Or am I missing some namespace somewhere?

Reply:
No, I rename a lemma, but because it's now in another file I deprecate the old lemma in place and use the new lemma to prove it (to show the type really hasn't changed)

Reply:
oh sorry :facepalm: I was reading it backwards.